sklearn/ensemble/_iforest.py

```diff
@@ -348,7 +348,9 @@ def fit(self, X, y=None, sample_weight=None):
 
         # Else, define offset_ wrt contamination parameter
         # To avoid performing input validation a second time we call
-        # _score_samples rather than score_samples
+        # X to a CSR matrix while it was validated earlier as a CSC matrix.
```

Comment:
I have a hard time parsing this. Perhaps

```
# To avoid performing input validation a second time we call
# _score_samples rather than score_samples.
# _score_samples expects a CSR matrix, so we convert if necessary.
```